internal/controller/suite_test.go

```diff
@@ -22,15 +22,15 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
-	nropv1 "github.com/openshift-kni/numaresources-operator/api/v1"
-	securityv1 "github.com/openshift/api/security/v1"
-	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
same here, onsi's imports are not seperated from k8s ones

Reply:
they should I'll check why not